src/outlet/ticketStorage.ts

```diff
@@ -244,9 +261,32 @@ export class TicketStorage {
 		let tokenData: DecodedTokenData
 
 		if (type === 'eas') {
-			this.easManager.loadFromEncoded(token)
+			const { fieldDefinition, idFields, easManager } = this.getEasManagerAndFieldLabelsForToken(token)
 
-			tokenData = this.easManager.getAttestationData() as DecodedTokenData
+			easManager.loadFromEncoded(token)
+
+			const easAttest = easManager.getEasJson()
+			const easData = easManager.getAttestationData() as any
+
+			const easFieldData = {}
+
+			for (const fieldDef of fieldDefinition) {
+				easFieldData[fieldDef.name] = {
+					label: fieldDef.label ?? fieldDef.name,
+					value: easData[fieldDef.name],
+				}
+			}
+
+			tokenData = {
+				eventId: easData.eventId,
+				ticketIdString: easData.ticketIdString,
```

Comment:
ticketIdString. This would relate to the comments in the other section of the PR around this naming convention / management of types.